tests/__main__.py

```diff
@@ -16,5 +16,5 @@
 
 opts = "-v -n=auto --dist loadscope --cov=dvc --durations=0"
 params = " ".join(sys.argv[1:])
-cmd = f"{sys.executable} -m pytest {opts} {params}"
+cmd = f"{sys.executable} -m pytest {opts} {params} tests"
```

Comment:
Was the omission of `tests` here intentional? Are there some relevant tests in `dvc` folder?